website/blog/2021/11/26/apache-apisix-committer-experience.md

```diff
@@ -0,0 +1,48 @@
+
+My interest lies in all things creative and **love to create products**, starting straight from bare conceptualisation till product completion and delivery, covering all things engineering and design in between.
+
+Apart from this I am an art and music connoisseur. Sometimes, you can find me listening to the same hiphop playlist for hours trying to make sense of every lyric and catching all the rhymes and double entendres in between. But enough about me, let's dwell into my journey as that's why you are reading probably (will try my best to keep it short)
+
+## Exploring beyond boundaries
+
+I have had a great time at Apache APISIX, working alongside highly cooperative, kind and humble people and feel writing down this journey of mine here would be a great starting point for anyone looking to enter this eternal space of Open Source with Apache APISIX. Though I believe everyone has their very own different journey and we can't just generalize, I will try to mention some points along that can help anyone irrespective, in easing their exploration struggles.
+
+So it all started from February, 2021. I was looking around GitHub to find OSS projects to contribute to, so as to use my knowledge for something that actually provides value to users. I would recommend people to check out the GSoC organisations page, try online searches or look for some GitHub repos that have a list of organisations for getting a starting point. Of Course you can contribute to projects randomly online, but I think this could be an efficient method. After looking around and testing multiple waters, I came across Apache APISIX. I looked up their JIRA and got in contact with [Zhiyuan Ju](https://github.com/juzhiyuan) (**PMC at Apache APISIX, and a great guy to work with**). After I got the slack invite from him to asf-slack, things started to get in motion. I picked up issues from the [apisix-website](https://github.com/apache/apisix-website) repository and worked along.
+
+## New perspective to APISIX
+
+While I was working on the website, I noticed that it was lacking on a lot of fronts and there was a wide gap between features that Apache APISIX provides and what was presented on the website. I discussed this with Zhiyuan and he agreed with my opinion. As [Google Summer of Code](https://summerofcode.withgoogle.com/archive/) was near, I proposed that I work on this project under this program. We both discussed this thoroughly and agreed on it. Through this project I met [Shuyang Wu](http://github.com/yiyiyimu), another awesome guy. Zhiyuan, Shuyang and Ming Wen were my mentors for this project. We started off in May and finally the new website was successfully up by September, 2021.
+
+![image](https://user-images.githubusercontent.com/57267960/143325043-acf485e1-2f07-4d3c-b693-debfd9fb788a.png)
```

Comment:
Could you upload this picture to https://markdown.apiseven.com/#/, and copy the link generated there, to replace this one?

Reply:
Done